video.go

```diff
@@ -80,7 +80,17 @@ func DefaultVideoOptions() VideoOptions {
 
 // MakeGIF takes a list of images (as frames) and converts them to a GIF.
 func MakeGIF(opts VideoOptions) *exec.Cmd {
-	if opts.Output.GIF == "" {
+
+	// Make a new variable so we don't have to overwrite
+	// config options.
+	var target_file string = opts.Output.GIF
```

Comment:
In Go, we generally use `camelCase` for variable names.

```suggestion
	var targetFile string = opts.Output.GIF
```